examples/graph/plot_16_Traid_Types.py

```diff
@@ -0,0 +1,50 @@
+"""
+======================
+16 Triad Types
+======================
```

Comment:
Just a nit, but the headline/underline length should match the length of the title.

Also, I'd prefer a more generic title, e.g just `Triads`, then move the explanation of the 16 triad types (and why there are 16) to the summary below.
```suggestion
==============
16 Triad Types
==============
```

Reply:
Thanks for the review Sir. I have made the changes. Hope it looks better :)

Reply:
Thanks for the review Sir. I have made the changes. Hope it looks better :)